packages/frontend/tsconfig.json

```diff
@@ -13,7 +13,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "paths": {
```

Comment:
What does this do?

Reply:
paths allows you not to use relative path.  The root of the dir can be accessable with ~~/pages
rather than ../../pages etc
This also allows typescript to correct and validate imports when you move files.

Reply:
Make sense. Thanks